test/gitea_test.exs

```diff
@@ -162,6 +162,26 @@ defmodule GiteaTest do
     Logger.error("test: local_branch_create/1 > error: #{error}")
   end
 
+  test "create_branch/3 and switch/3 create a new branch and switch" do
+    org_name = "myorg"
+    repo_name = create_test_git_repo(org_name)
+    {:ok, branch_name} = Git.branch(%Git.Repository{path: @cwd}, ~w(--show-current))
+    Git.branch(Gitea.Helpers.local_git_repo(org_name, repo_name), ~w(-D draft2))
+
+    assert {:ok, res} = Gitea.create_branch(org_name, repo_name, "draft2")
```

Comment:
These warnings are annoying. putting an underscore in the pattern match of an `assert` doesn't work. 🤦‍♂️ 
```suggestion

    assert {:ok, res} = Gitea.create_branch(org_name, repo_name, "draft2")
```